lldb/source/Core/ValueObject.cpp

```diff
@@ -615,7 +615,16 @@ bool ValueObject::GetSummaryAsCString(TypeSummaryImpl *summary_ptr,
       m_synthetic_value->UpdateValueIfNeeded(); // the summary might depend on
                                                 // the synthetic children being
                                                 // up-to-date (e.g. ${svar%#})
-    summary_ptr->FormatObject(this, destination, actual_options);
+
+    TargetSP target = GetExecutionContextRef().GetTargetSP();
```

Comment:
`s/target/target_sp/`

We try to make sure we append `_sp` for any variables that are shared pointers and `_up` for any variables that are unique pointers.